### PR TITLE
Create table for industries #67

### PR DESCRIPTION
**Pull request by OlesiaBarannik**

### CODE REVIEW

File: projects/serializers.py [hunk 1]
```diff
@@ -40,15 +50,15 @@ def validate(self, data):
...
+        #     industries = [industry[0] for industry in Project.INDUSTRY_CHOICES]
+        #     if industry not in industries:
+        #         raise serializers.ValidationError(
+        #             {
+        #                 "status": "failed",
+        #                 "message": "Industry does not exist",
+        #                 "choices": industries
+        #             }
+        #         )
         return data
```
Comment (NovitskaMariia) [inline projects/serializers.py:64]: If this code we would'n use in future, maybe we can delete it? (commented code)

**Review by NovitskaMariia — APPROVED** [review]
All methods works well. Table for industries filling successfully.

File: startups/serializers.py [hunk 2]
```diff
@@ -28,6 +29,7 @@ class Meta:
         read_only_fields = ('registration_date', 'owner')
 
     def create(self, validated_data):
+        print(validated_data)
         user = self.context['request'].user
```
Comment (Thoughtseize1) [inline startups/serializers.py:33]: I do not think this `print `is necessary.

**Review by Thoughtseize1 — APPROVED** [review]
It all works. 
Tables are being created.
When creating a startup, the ID industry enters the table as intended. 
I have added a small comment to remove the extra print, at your discretion.

File: startups/models.py [hunk 3]
```diff
@@ -2,15 +2,23 @@
...
 class Startup(models.Model):
     id = models.AutoField(primary_key=True)
     owner = models.ForeignKey(CustomUser, related_name='owner', on_delete=models.CASCADE)
     startup_name = models.CharField(max_length=255, unique=True)
     description = models.TextField(blank=True)
-    industries = models.TextField(blank=True)
+    industries = models.ForeignKey(Industry, on_delete=models.CASCADE, related_name='startups', null=True, blank=True)
     location = models.CharField(max_length=255, blank=True)
-    contact_phone = models.CharField(max_length=128, blank=True)
+    contact_phone = models.CharField(max_length=128, unique=True)
```
Comment (VovaD199) [inline startups/models.py:21]: a cell phone doesn't have to be unique, we talked about that before, but leave it at that

**Review by VovaD199 — APPROVED** [review]

File: projects/serializers.py (hunk 1 above)
Comment (OlesiaBarannik) [reply projects/serializers.py:64]: Ok, thank you

File: startups/serializers.py (hunk 2 above)
Comment (OlesiaBarannik) [reply startups/serializers.py:33]: done

File: startups/models.py (hunk 3 above)
Comment (OlesiaBarannik) [reply startups/models.py:21]: Ok, thank you

**Pull request merged**